### ISSUE TITLE
Update the eslint dependency.

### ISSUE THREAD

**Issue by metal-gogo** [gbmdigital/javascript#5]
The eslint dependency is set with the version 2.10, but it's a little bit old version of [the package](https://www.npmjs.com/package/eslint)

**Comment by luismendozamx** [comment]
@metal-gogo I agree, we can upgrade whenever. Can you make a PR so we can test against the  repos that are using this dependency?
